packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -230,20 +230,19 @@ export default class TSGenerator extends CodeGenerator {
         let filePath = sourceFile.getFilePath().toString();
         filePath = filePath.substring(1);
 
+        /**
+         * It's not Prettier-level of nice but  `ts-morph` offers a method of using the TS
+         * formatter for formatting code which we can use to make our generated SDK not look like
+         * total garbage.
+         *
+         * @see {@link https://ts-morph.com/manipulation/formatting}
+         */
+        sourceFile.formatText();
```

Comment:
nice, TIL! kinda curious about the performance of this vs prettier vs others (but don't think we need to investigate that any time soon)